catalogs/aub.yaml

```diff
@@ -67,6 +71,7 @@ sources:
       set: "travelbooks"
     metadata:
       data_path: aub/travelbooks
+      config: aub_travelbooks_config_csv
```

Comment:
naive question: what's `config` here reference/where's it get used?  e.g. if i grep `cambridge_islamic_config` and `manchester_nashriyah_config_csv`, the only matches i come up with are in these config files.